### PR TITLE
Adding one apply UC and workspace example

### PR DESCRIPTION
**Pull request by flaviomalavazi**
Changes:
- Creating AWS and Databricks resources to build a simple workspace with UC enabled
- Creating Databricks account level groups, promoting them to UC Metastore owners and adding permissions

### CODE REVIEW

**Review by RodrigoLima82 — APPROVED** [review]

**Review by alexott — CHANGES_REQUESTED** [review]
put modules into the top-level `modules` folder, so they could be visible in the Terraform registry UI.

**Comment by flaviomalavazi** [comment]
> put modules into the top-level `modules` folder, so they could be visible in the Terraform registry UI.

Hey, @alexott, I've moved the modules to the top level folder, as well as updated the top level readme documentation and bumped provider versions

**Review by nfx — APPROVED** [review]

File: modules/aws-databricks-unity-catalog/versions.tf [hunk 1]
```diff
@@ -0,0 +1,15 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "=4.57.0"
```
[the remark this answers is not in the document]
Comment (flaviomalavazi) [reply modules/aws-databricks-unity-catalog/versions.tf]: Adjusted provider version to >=4.57

File: modules/aws-databricks-base-infra/versions.tf [hunk 2]
```diff
@@ -0,0 +1,14 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "=4.57.0"
```
[the remark this answers is not in the document]
Comment (flaviomalavazi) [reply modules/aws-databricks-base-infra/versions.tf]: Adjusted provider version to >=4.57

File: examples/aws-workspace-uc-simple/databricks_unity_catalog.tf [hunk 3]
```diff
@@ -0,0 +1,11 @@
+resource "databricks_grants" "unity_catalog_grants" {
+  provider  = databricks.workspace
+  metastore = module.unity_catalog.metastore_id
+  grant {
+    principal  = local.unity_admin_group
```
[the remark this answers is not in the document]
Comment (flaviomalavazi) [reply examples/aws-workspace-uc-simple/databricks_unity_catalog.tf]: Done!

File: examples/aws-workspace-uc-simple/databricks_clusters.tf [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    min_workers = 1
+    max_workers = 5
+  }
+
+  spark_conf = {
+    "spark.databricks.io.cache.enabled" : true,
+    "spark.databricks.io.cache.maxDiskUsage" : "50g",
+    "spark.databricks.io.cache.maxMetaDataCache" : "1g",
+    "spark.databricks.unityCatalog.userIsolation.python.preview" : true
+  }
```
[the remark this answers is not in the document]
Comment (flaviomalavazi) [reply examples/aws-workspace-uc-simple/databricks_clusters.tf]: Removed

**Comment by flaviomalavazi** [comment]
Hey, @alexott, all good here?

**Pull request merged**